docs_nnx/guides/bridge_guide.md

```diff
@@ -415,77 +524,78 @@ print(var['params'])
 
-Even if you are not using any partition metadata in your Linen module, the variable JAX arrays will be converted to `nnx.Variable`s that wraps the true JAX array within.
+> **Note:** To get more familiarized with sharding metadata with Flax and JAX, refer to Flax NNX’s [Scale up](https://flax.readthedocs.io/en/latest/guides/flax_gspmd.html) guide, JAX’s [Introduction to parallel programming](https://jax.readthedocs.io/en/latest/sharded-computation.html#sharded-computation), and the Flax Linen [Scale up](https://flax-linen.readthedocs.io/en/latest/guides/parallel_training/flax_on_pjit.html) guide.
 
-If you use `nn.with_partitioning` to annotate your Linen module's variables, the annotation will be converted to a `.sharding` field in the corresponding `nnx.Variable`.
+### Partition metadata: Flax Linen to NNX
 
-You can then use `nnx.with_sharding_constraint` to explicitly put the arrays into the annotated partitions within a `jax.jit`-compiled function, to initialize the whole model with every array at the right sharding.
+Even if you are not using any partition metadata in your Flax Linen `Module`, the variable [`jax.Array`s](https://jax.readthedocs.io/en/latest/key-concepts.html#jax-arrays-jax-array) will be converted to [`nnx.Variable`s](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/variables.html#flax.nnx.Variable) that wraps the true `jax.Array` within.
 
+If you use [`flax.linen.with_partitioning`](https://flax-linen.readthedocs.io/en/latest/api_reference/flax.linen/spmd.html#flax.linen.with_partitioning) (`nn.with_partitioning`) to annotate your Flax Linen `Module` variables, the annotation will be converted to the `.sharding` field in the corresponding [`nnx.Variable`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/variables.html#flax.nnx.Variable).
 
-```python
+You can then use [`nnx.with_sharding_constraint`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/spmd.html#flax.nnx.with_sharding_constraint) to explicitly put the arrays into the annotated partitions within a [`jax.jit`](https://jax.readthedocs.io/en/latest/jit-compilation.html)-compiled function, to initialize the whole model with every array at the right sharding.
+
+```{code-cell} ipython3
 class LinenDotWithPartitioning(nn.Module):
   out_dim: int
   @nn.compact
   def __call__(self, x):
-    w = self.param('w', nn.with_partitioning(nn.initializers.lecun_normal(),
```

Comment:
Since the code was updated, will revert back to the new recent version @cgarciae